roles/galaxy-route/tasks/main.yml

```
@@ -15,6 +15,27 @@
     - ingress_type is defined
     - ('route' == ingress_type|lower) or ('ingress' == ingress_type|lower)
 
+- name: Remove deprecated {{ deployment_type }} Routes on OpenShift for {{ ansible_operator_meta.name }} from {{ ansible_operator_meta.namespace }} namespace
+  kubernetes.core.k8s:
+    state: absent
+    api_version: v1
+    kind: Route
+    namespace: "{{ ansible_operator_meta.namespace }}"
+    name: "{{ item }}"
+  loop:
+    - "{{ ansible_operator_meta.name }}-ansible-pulp-ansible-galaxy"
+    - "{{ ansible_operator_meta.name }}-api-v3"
+    - "{{ ansible_operator_meta.name }}-auth"
+    - "{{ ansible_operator_meta.name }}-container-extensions-v2"
+    - "{{ ansible_operator_meta.name }}-container-pulp-container"
+    - "{{ ansible_operator_meta.name }}-container-token"
+    - "{{ ansible_operator_meta.name }}-container-v2"
+    - "{{ ansible_operator_meta.name }}-content"
+    - "{{ ansible_operator_meta.name }}-galaxy-api"
+    - "{{ ansible_operator_meta.name }}-galaxy-ui"
+    - "{{ ansible_operator_meta.name }}-galaxy-ui2"
+  when: is_openshift
```

For consistency with the rest of the code we should rather use
```suggestion
  when: ingress_type | lower == 'route'
```